judges/pipeline.py

```diff
@@ -1,13 +1,206 @@
 """Python script responsible for extracting judge data using web scraping"""
 
 from os import environ as ENV
-from time import sleep
 import logging
+from datetime import datetime
 from dotenv import load_dotenv
 import requests
 from bs4 import BeautifulSoup
+import pandas as pd
+from psycopg2 import connect
+from psycopg2.extras import RealDictCursor
 
 
-if __name__ == "__main__":
+# ========== GLOBALS ==========
+HCKB_URL = "https://www.judiciary.uk/about-the-judiciary/who-are-the-judiciary/senior-judiciary-list/kings-bench-division-judges/"
+CJ_URL = "https://www.judiciary.uk/about-the-judiciary/who-are-the-judiciary/list-of-members-of-the-judiciary/circuit-judge-list/"
```

Comment:
Maybe `KINGS_BENCH_URL` and `CIRCUIT_JUDGE_URL` instead of the current abbreviated names? Just so its more clear what these are.

Reply:
I see what you mean, if we do expand all the abbreviations to full names though, lines like `HCKB = scrape_high_court_kings_bench(HCKB_URL)` might be quite long, so not sure which ones to abbreviate which not

Reply:
Instead of calling it `high_court_kings_bench` you could just call it `kings_bench` or `high_court_judges` so it's a bit shorter. I just think it needs to be a little bit more descriptive than `HCKB` and `CJ`.